.dev/docker-compose.yaml

```diff
@@ -13,18 +13,18 @@ services:
   leantime-dev:
     build: .
     ports:
-      - "8001:8080" #Using ports that are hopefully available. format is: 80XX
+      - "8888:8080" #Using ports that are hopefully available. format is: 80XX
```

Comment:
Can you revert the changes to this file?

Reply:
done